packages/stryker-karma-runner/src/starters/stryker-karma.conf.ts

```diff
@@ -15,13 +15,17 @@ function setDefaultOptions(config: Config) {
 function setUserKarmaConfigFile(config: Config, log: Logger) {
   if (globalSettings.karmaConfigFile && typeof globalSettings.karmaConfigFile === 'string') {
     const configFileName = path.resolve(globalSettings.karmaConfigFile);
-    log.debug('Importing config from "%s"', configFileName);
     try {
       const userConfig = requireModule(configFileName);
+      log.debug('Importing config from "%s"', configFileName);
```

Comment:
Is there a reason why you moved this logging statement?

Reply:
I wanted to test it, but I didn't manage to do it :(

Reply:
Yes, I think it would be smart to still log that we're going to try to read the config file

Reply:
Sure. Changing it back right now